fe/fe-core/src/main/java/org/apache/doris/clone/TabletScheduler.java

```diff
@@ -826,12 +831,25 @@ private boolean deleteReplicaNotInCluster(TabletSchedCtx tabletCtx, boolean forc
         return false;
     }
 
+    private boolean deleteSrcReplicaOfLB(TabletSchedCtx tabletCtx, boolean force) throws SchedException {
```

Comment:
What's the meaning of `LB` ？

Reply:
LoadBalance，I think the naming should be changed too.